messaging/src/main/java/org/jboss/as/messaging/TransportConfigOperationHandlers.java

```diff
@@ -145,6 +145,13 @@ static void processConnectors(final OperationContext context, final Configuratio
                 final ModelNode config = property.getValue();
                 final Map<String, Object> parameters = getParameters(context, config);
                 final String clazz = FACTORY_CLASS.resolveModelAttribute(context, config).asString();
+                //we treat it as remote_connector
+                ModelNode bindingNode = config.get(GenericTransportDefinition.SOCKET_BINDING.getName());
+                if (bindingNode != null && bindingNode.isDefined()) {
+                    final String binding = bindingNode.asString();
+                    parameters.put(RemoteTransportDefinition.SOCKET_BINDING.getName(), binding);
```

Comment:
for consistency sake, let's also use `GenericTransportDefinition.SOCKET_BINDING` here :)